README.md

```diff
@@ -62,172 +53,22 @@ The new registry implementation provides the following benefits:
 
 For information on upcoming functionality, please see [ROADMAP.md](ROADMAP.md).
 
-Installation
-------------
+Deploying
```

Comment:
I got some feedback that the flow of this readme was a little sudden. We may want to add a heading here, indicating that the following sections apply to the registry.

@bfirsh